Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "PyCall"
+uuid = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
+authors = ["Steven G. Johnson <stevenj@mit.edu>", "Yichao Yu <yyc1992@gmail.com>", "Takafumi Arakaki <aka.tkf@gmail.com>", "Simon Kornblith <simon@simonster.com>", "Páll Haraldsson <Pall.Haraldsson@gmail.com>", "Jon Malmaud <malmaud@gmail.com>", "Jake Bolewski <jakebolewski@gmail.com>", "Keno Fischer <keno@alumni.harvard.edu>", "Joel Mason <jobba1@hotmail.com>", "Jameson Nash <vtjnash@gmail.com>", "The JuliaPy development team"]
+version = "1.91.2-DEV"
```

Comment:
`version` is somewhat random, too

Reply:
I'm confused by the release process with a Project.toml.   Do we update the `version` and then tag a release?  In which case the `version` here should be the last released/tagged version?

Reply:
I think this is a good quick summary https://discourse.julialang.org/t/22677/32

But first you need to install the Registrator app https://github.com/apps/registratortest/installations/new (see https://github.com/JuliaComputing/Registrator.jl)

> Do we update the `version` and then tag a release?

I think we update `version` in Project.toml and comment ```@JuliaRegistrator `register()```` in github.  [Adding a Git tag is optional](https://discourse.julialang.org/t/switching-package-registration-systems-on-monday/22677/20) but I think it's better to do that.

> In which case the `version` here should be the last released/tagged version?

Yeah, I think that's a valid option too.  I'll change it.